workshops/templates/workshops/event_edit_form.html

```diff
@@ -4,32 +4,68 @@
 {% load selectable_tags %}
 
 {% block content %}
-  <button type="button" class="btn btn-default" id="update_url_button">Update from URL</button>
-  {% crispy event_form %}
-
-  {% include "workshops/event_import_update_from_url.html" with update=True %}
-
-  <h2 id="associated-tasks">Associated tasks</h2>
-  {% crispy task_form form_helper_with_add %}
-  {% if tasks %}
-  <table class="table table-striped">
-  <tr>
-    <th>person</th>
-    <th>role</th>
-    <th>actions</th>
-  </tr>
-  {% for t in tasks %}
-  <tr>
-    <td><a href="{{ t.person.get_absolute_url }}">{{ t.person.get_full_name }}</a>{% if t.person.email and t.person.may_contact %} &lt;{{ t.person.email|urlize }}&gt;{% endif %}</td>
-    <td>{{ t.role.name }}</td>
-    <td><a onclick='return confirm("Are you sure you wish to remove \"{{ t.person.get_full_name }}\" from {{ object.get_ident }}?")' href="{% url 'task_delete' object.get_ident t.id %}" class='btn btn-danger'>Delete</a></td>
-  </tr>
-  {% endfor %}
-  </table>
-  {% else %}
-  <p>No tasks.</p>
-  {% endif %}
-
+<div>
+  <ul class="nav nav-tabs" role="tablist">
+    <li role="presentation" class="active"><a href="#event" aria-controls="event" role="tab" data-toggle="tab">Event</a></li>
+    <li role="presentation"><a href="#tasks" aria-controls="tasks" role="tab" data-toggle="tab">Tasks</a></li>
+    <li role="presentation"><a href="#sponsors" aria-controls="sponsors" role="tab" data-toggle="tab">Sponsors</a></li>
```

Comment:
See how `person-edit` page behaves after you add a new task to the person -- it switches you to a tab you just were at.

You'll receive 1000 bonus virtual points if you make the tabs switch depending on the anchor in the browser's address bar. :-)